### DISCUSSION TITLE
License for VerneMQ

### DISCUSSION THREAD

**Discussion by uk1988** [vernemq/vernemq#2070] [Q&A]
Hi All

How can you acquire license for VerneMQ ? 
We have recently tried to contact Verne over the page https://vernemq.com/contact.html but we got no response. 

Best Uros

**Comment by ioolkos** [comment]
@uk1988 I will look for your ticket and take care of it. [sometimes stuff gets classified wrongly or mixed up in Helpscout].
Thank you for the reminder and apologies.

**Comment by ioolkos** [comment]
@uk1988 I can't seem to identify your ticket. Can you resend a quick note via the contact page?

**Comment by uk1988** [comment]
@ioolkos  just send it as Uros Kolaric
